Source/WebKit/Scripts/webkit/tests/MessageNames.h

```diff
@@ -291,6 +302,36 @@ constexpr bool messageIsSync(MessageName name)
     return name >= MessageName::FirstSynchronous;
 }
 
+constexpr ASCIILiteral processLiteral(ProcessName name)
+{
+    switch (name) {
+    case ProcessName::UI:
+        return "UI";
+    case ProcessName::Networking:
+        return "Networking";
+    case ProcessName::GPU:
+        return "GPU";
+    case ProcessName::WebContent:
+        return "WebContent";
+    case ProcessName::Model:
+        return "Model";
+    case ProcessName::Unknown:
+        return "Unknown";
+    default:
+        RELEASE_ASSERT_NOT_REACHED();
+    }
+}
```

Comment:
This doesn't all need to be in a header that's included everywhere.  That'll slow down the build.  Just have function declarations here and the bulk in the cpp.  It's only used for debug builds, so it doesn't need to be constexpr.

Reply:
Done.